<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="summa">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-color: white;
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border-bottom: 1px solid gray;
}
background-color : white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ac">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>56</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: bold;
	background-color: #00FFFF;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid blue;
	background-color: #FFE4E1;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plmin">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>56</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: bold;
	background-color: #00FFFF;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid blue;
	background-color: #FFE4E1;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>56</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: bold;
	background-color: #00FFFF;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid blue;
	background-color: #FFE4E1;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_baj">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>56</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: bold;
	background-color:  #D8BFD8;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid black;
	background-color: #FFFFE0	;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>121</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: bold;
	background-color: #F0FFF0;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid #00FF00;
	background-color: #98FB98;
}
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>121</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: bold;
	background-color: #F0FFF0;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid #00FF00;
	background-color: #98FB98;
}
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>121</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: bold;
	background-color: #F0FFF0;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid #00FF00;
	background-color: #98FB98;
}
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_baz">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>121</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: bold;
	background-color:  #D8BFD8;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid black;
	background-color: #FFFFE0	;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>186</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: bold;
	background-color: #F0FFF0;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid #00FF00;
	background-color: #98FB98;
}
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>186</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: bold;
	background-color: #F0FFF0;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid #00FF00;
	background-color: #98FB98;
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>186</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: bold;
	background-color: #F0FFF0;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid #00FF00;
	background-color: #98FB98;
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>186</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: bold;
	background-color:  #D8BFD8;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid black;
	background-color: #FFFFE0	;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>251</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: bold;
	background-color: #F0FFF0;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid #00FF00;
	background-color: #98FB98;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>251</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: bold;
	background-color: #F0FFF0;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid #00FF00;
	background-color: #98FB98;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>251</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: bold;
	background-color: #F0FFF0;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{	
	border: 2px solid #00FF00;
	background-color: #98FB98;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pl">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>251</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: bold;
	background-color:  #D8BFD8;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid black;
	background-color: #FFFFE0	;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>316</y>
      <width>128</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	font: bold;
	background-color: #F0FFF0;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid #00FF00;
	background-color: #98FB98;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_hav">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>316</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: bold;
	background-color:  #D8BFD8;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid black;
	background-color: #FFFFE0	;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>316</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: bold;
	background-color: #00FFFF;
	border: 2px solid gray;
	border-radius: 10px;
}
QPushButton:hover
{
	border: 2px solid blue;
	background-color: #FFE4E1;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
